CONTRIBUTING.rst

```diff
@@ -106,6 +106,17 @@ After talking to the Satpy developers any additional work like code or
 documentation changes can be provided as a GitHub
 `Pull Request <https://help.github.com/articles/creating-a-pull-request/>`_.
 
+To make sure that your code complies with the pytroll python standard, you can
+run the `flake8 <http://flake8.pycqa.org/en/latest/>`_ linter on you change
```

Comment:
"your changes before you submit them"?

Reply:
```suggestion
run the `flake8 <http://flake8.pycqa.org/en/latest/>`_ linter on your changes
```

Reply:
The next sentence says "submit it" which should be "submit them" with your suggested changes.